Fix #4508 - update docs for -ShellId (#4509)
MicrosoftDocs/PowerShell-Docs: reference/3.0/Microsoft.PowerShell.Core/About/about_Requires.md

# About Requires

## Short description
Prevents a script from running without the required elements.

## Long description

The `#Requires` statement prevents a script from running unless the

> PowerShell version, modules, snap-ins, module, and snap-in version

Reviewer: ```suggestion
PowerShell version, modules (and version), or snap-ins (and version)
```
